core/src/main/java/org/elasticsearch/index/query/ExistsQueryBuilder.java

```diff
@@ -74,19 +74,23 @@ public QueryValidationException validate() {
     }
 
     public static Query newFilter(QueryParseContext parseContext, String fieldPattern) {
```

Comment:
this could be marked as deprecated? or maybe //norelease ? should be removed when we are done with the refactoring right?